packages/next/src/build/babel/loader/get-config.ts

```diff
@@ -325,6 +329,10 @@ function getFreshConfig(
   }
 
   if (loaderOptions.transformMode === 'standalone') {
+    if (!reactCompilerPluginsIfEnabled.length) {
```

Comment:
will this affect other user customized config? if we can have a test to demostrate the change would be great

Reply:
It will not be affected because of `if (loaderOptions.transformMode === 'standalone')`